create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py

```diff
@@ -78,6 +81,19 @@ def init_jinja2(app: web.Application) -> None:
     await app['redis_sub'].wait_closed()
     await app['redis_pub'].wait_closed()
 {%- endif %}
+{%- if cookiecutter.use_sentry == 'y' %}
+
+
+def init_sentry(app: web.Application) -> None:
+    '''
+    Initialize sentry sdk for aiohttp
+    '''
+    config = app['config']['sentry']
+    sentry_sdk.init(
+        dsn=f'https://{configp["key"]}@sentry.io/{config["project"]}',
```

Comment:
Nope. User should provide the complete DSN via the config. Not a separate key. They may have an on-premise install of it and so on. Don't steal the control of this from them.

Reply:
Chaged for the dsn key in config template.